usaspending_api/disaster/v2/views/object_class/object_class_result.py

```diff
@@ -30,9 +30,13 @@ def rollup(self):
             for child in row.children:
                 row.outlay += child.outlay
                 row.obligation += child.obligation
-                row.count += child.count
                 row.total_budgetary_resources += child.total_budgetary_resources
 
+                if child.award_count:
+                    row.award_count += child.award_count
+                else:
+                    row.award_count = None
```

Comment:
Hopefully the edge-case of a child record having a count < 1 isn't encountered. Otherwise the code could cause an exception.